src/main/scala/tile/BaseTile.scala

```diff
@@ -146,6 +146,10 @@ abstract class BaseTile(tileParams: TileParams, val crossing: ClockCrossingType)
   protected val tlSlaveXbar = LazyModule(new TLXbar)
   protected val intXbar = LazyModule(new IntXbar)
 
+  val traceNode = BundleBridgeSource(() => Vec(tileParams.core.retireWidth, new TracedInstruction()))
+  //val traceNode = BundleBroadcast[Vec[TracedInstruction]]
+  //traceNode := traceSourceNode
```

Comment:
I think switching to the Broadcast version is necessary, because otherwise diplomacy will error out for configurations that have no further nodes connected to the BundleBridgeSource